golos.publication/golos.publication.cpp

```diff
@@ -592,8 +594,10 @@ void publication::set_vote(name voter, name author, string permlink, int16_t wei
     std::vector<structures::delegate_voter> delegators;
     auto token_code = pool->state.funds.symbol.code();
     auto list_delegate_voter = golos::vesting::get_list_delegate(config::vesting_name, voter, token_code);
+    auto effective_vesting = golos::vesting::get_account_effective_vesting(config::vesting_name, voter, token_code);
     for (auto record : list_delegate_voter) 
-        delegators.push_back( {record.sender, record.quantity} );
+        delegators.push_back( {record.sender, record.quantity, record.interest_rate, 
+                record.payout_strategy, effective_vesting} );
```

Comment:
Why do you save effective_vesting for any delegate record? As I can see it depends from voter only and it's value equal for any delegator. So, maybe it's better to save it in vote_object field?